spec/controllers/api/v1/restaurants_controller_spec.rb

```diff
@@ -1,26 +1,35 @@
 require "rails_helper"
-require "json"
+
 
 RSpec.describe Api::V1::RestaurantsController, type: :controller do
-  describe "GET#index" do
```

Comment:
The test for the index action was removed?

Reply:
fang had us remove it yesterday.  cause of the way our 3rd party api that does the initial request.